src/exercise/05.js

```diff
@@ -12,16 +12,37 @@ import '../box-styles.css'
 
 // 🐨 add a style prop to each of them as well so their background color
 // matches what the text says it should be as well as `fontStyle: 'italic'`
-const smallBox = <div>small lightblue box</div>
-const mediumBox = <div>medium pink box</div>
-const largeBox = <div>large orange box</div>
+const Box = (props) => {
+  const { size, style, ...children } = props;
+  return (
+    <div 
+      className={`box box--${size}`}
+      style={{ fontStyle: 'italic', ...style}}
+      {...children}
+    />)
+}
 
 function App() {
   return (
     <div>
-      {smallBox}
-      {mediumBox}
-      {largeBox}
+      <Box 
+        size="small" 
+        style={{ backgroundColor: 'lightblue' }}
+      >
+        small lightblue box
```

Comment:
Not sure why my text didn't show up when doing it this way so I added another prop. I think its cleaner like this thoug.